fizzbuzz.rb

```diff
@@ -10,3 +10,12 @@
 #   otherwise
 #       return the original number (as a string)
 
+def fizz_buzz(number)
+  if(number == 3)
```

Comment:
**suggestion:** If this function keeps growing, then it might be worth refactoring and separating it out in smaller functions. For example, a function that checks divisibility with 3 and another for divisibility with 5.